apps/web/pages/event-types/index.tsx

```diff
@@ -128,17 +125,11 @@ const Item = ({ type, group, readOnly }: { type: EventType; group: EventTypeGrou
           data-testid={"event-type-title-" + type.id}>
           {type.title}
         </span>
-        {group.profile.slug ? (
-          <small
-            className="hidden font-normal leading-4 text-gray-600 sm:inline"
-            data-testid={"event-type-slug-" + type.id}>
-            {`/${group.profile.slug}/${type.slug}`}
-          </small>
-        ) : (
-          <></>
-        )}
+        <small
+          className="hidden font-normal leading-4 text-gray-600 sm:inline"
+          data-testid={"event-type-slug-" + type.id}>{`/${group.profile.slug}/${type.slug}`}</small>
         {readOnly && (
-          <span className="inline items-center rounded-sm bg-gray-100 px-1.5 py-0.5 text-xs font-medium text-gray-800 ltr:ml-2 ltr:mr-2 rtl:ml-2">
+          <span className="rtl:ml-2inline items-center rounded-sm bg-gray-100 px-1.5 py-0.5 text-xs font-medium text-gray-800 ltr:ml-2 ltr:mr-2">
```

Comment:
same here. also typo

```suggestion
          <span className="rtl:ml-2 inline items-center rounded-sm bg-gray-100 px-1.5 py-0.5 text-xs font-medium text-gray-800 ltr:ml-2 ltr:mr-2">
```

Reply:
the span tag is already inline, it is redundant